testsuite/gnat2goto/tests/type_modular_non_power_of_two_decl/modular_type.adb

```diff
@@ -1,25 +1,19 @@
 procedure Modular_Type is
   type Less_Than_Ten is mod 10;
   procedure Assert_Val (X : Less_Than_Ten) is
-     Y : Less_Than_Ten := X;
-     Z : Less_Than_Ten := Y;
+     Y : Less_Than_Ten := X - 1;
+     Z : Less_Than_Ten := X;
```

Comment:
:pick: Seems unnecessary, why not just use `X` instead?

Reply:
Done.